### PR TITLE
Parse bio content

### PR DESCRIPTION
**Pull request by martindsq**
#923 

Re-uses the code to parse notes in BioView.
Updates the style and spacing to match the design.
Update the text style in the view the user sees when tapping on the Bio to see the full Bio.

### CODE REVIEW

File: Nos/Views/BioView.swift [hunk 1]
```diff
@@ -32,9 +34,25 @@ struct BioView: View {
...
 
+    private var parsedBio: AttributedString {
+        guard let bio else {
+            return AttributedString()
+        }
+        return NoteParser.parse(
+            content: bio,
+            tags: [[]],
+            context: viewContext
+        )
```
Comment (joshuatbrown) [inline Nos/Views/BioView.swift:45]: @martindsq as I was testing this, I noticed that we don't detect links in the bio for text like `nos.social`. I dug into why and found that the _other_ parse method (the one that returns `(AttributedString, [URL])`) calls extractURLs but _this_ parse method doesn't. Could we add extractURLs to this one (in NoteParser) so we get the updated link detection?

**Review by joshuatbrown — CHANGES_REQUESTED** [review]
Looks great! One request.

File: Nos/Views/BioView.swift (hunk 1 above)
Comment (martindsq) [reply Nos/Views/BioView.swift]: Good catch, I didn't realize we have two nearly identical functions

Comment (martindsq) [reply Nos/Views/BioView.swift]: I created a new ticket https://github.com/planetary-social/nos/issues/928

**Review by joshuatbrown — APPROVED** [review]
Thanks for removing unused code! Looks great!

**Pull request merged**